packages/shared/src/components/search/SearchPanel/SearchPanelPostSuggestions.tsx

```diff
@@ -35,7 +35,7 @@ const PanelItem = ({
     <SearchPanelItem {...rest} icon={<SearchIcon />} {...itemProps}>
       {!!purifySanitize && (
         <span
-          className="text-theme-label-tertiary typo-subhead [&>strong]:text-white"
+          className="typo-subhead [&>strong]:text-white"
```

Comment:
First time even seeing this selector 🤯

Reply:
Right? 😆 I was surprised you can do it in tailwind I almost went to raw css file..